Add FreeBSD and OpenBSD to release builds

It looks like all that would be required is adding "freebsd openbsd" to the end of the OS_PLATFORM_ARG list in cross binary script. Based on loose testing this seems fine, I tried NetBSD as well, but it gets a compile error from one of the dependencies.

Sure

```
---> Making bundle: crossbinary (in .)
Building binary for linux/386...
Building binary for linux/amd64...
Building binary for darwin/386...
Building binary for darwin/amd64...
Building binary for windows/386...
Building binary for windows/amd64...
Building binary for freebsd/386...
Building binary for freebsd/amd64...
Building binary for openbsd/386...
Building binary for openbsd/amd64...
Building binary for netbsd/386...
# github.com/containous/traefik/vendor/github.com/docker/engine-api/client
vendor/github.com/docker/engine-api/client/client.go:63: undefined: DefaultDockerHost
make: *** [crossbinary] Error 2
```

Looks like its just "netbsd" missing from https://github.com/docker/engine-api/blob/master/client/client_unix.go#L1

--- a/script/crossbinary
+++ b/script/crossbinary
@@ -22,7 +22,7 @@ fi
 rm -f dist/traefik_*
 
 # Build 386 amd64 binaries
-OS_PLATFORM_ARG=(linux darwin windows freebsd)
+OS_PLATFORM_ARG=(linux darwin windows freebsd openbsd)
 OS_ARCH_ARG=(386 amd64)
 for OS in ${OS_PLATFORM_ARG[@]}; do
   for ARCH in ${OS_ARCH_ARG[@]}; do

--- a/script/crossbinary
+++ b/script/crossbinary
@@ -22,7 +22,7 @@ fi
 rm -f dist/traefik_*
 
 # Build 386 amd64 binaries
-OS_PLATFORM_ARG=(linux darwin windows)
+OS_PLATFORM_ARG=(linux darwin windows freebsd)
 OS_ARCH_ARG=(386 amd64)
 for OS in ${OS_PLATFORM_ARG[@]}; do
   for ARCH in ${OS_ARCH_ARG[@]}; do

